internal/validation/validation.go

```diff
@@ -37,7 +37,7 @@ const (
 	containNumberRegexString          = `[0-9]`
 	containSpecialCharRegexString     = `[\{\}\[\]\/?.,;:|\)*~!^\-_+<>@\#$%&\\\=\(\'\"\x60]`
 	alphaNumberSpecialCharRegexString = `^[a-zA-Z0-9\{\}\[\]\/?.,;:|\)*~!^\-_+<>@\#$%&\\\=\(\'\"\x60]+$`
-	timeDurationFormatRegexString     = `^(\d{1,2}h\s?)?(\d{1,2}m\s?)?(\d{1,2}s)?$`
+	timeDurationFormatRegexString     = `^(\d{1,2}h\s?)(\d{1,2}m\s?)?(\d{1,2}s)?$`
```

Comment:
`timeDurationFormatRegexString` is used to check whether the given string for `duration` is valid, so inserting the `1h` check seems to be out of the scope of the variable.

Responsibility for setting it is up to the user, so how about trusting them to set the appropriate values ​​for now?

Reply:
> `timeDurationFormatRegexString` is used to check whether the given string for `duration` is valid, so inserting the `1h` check seems to be out of the scope of the variable.

Yes, checking minimum limit on duration string format regex is not appropriate. It is tricky where to put limit checks...

> Responsibility for setting it is up to the user, so how about trusting them to set the appropriate values ​​for now?

Project settings will be used for application developers, or administrators. Maybe trusting them is right.
Reverted minimum limit related codes.